api/docker-compose.yaml

```diff
@@ -73,9 +73,22 @@ services:
     volumes:
       - gradle-cache:/.gradle
       - .:/w:cached
-    command: wait-for db:3306 -- ./run-migrations.sh schema
+    command: wait-for db:3306 -- ./generate-cdr/init-new-cdr-db.sh --cdr-db-name cdr
     env_file:
       - db-cdr/vars.env
+  db-public-migration:
```

Comment:
These db services are basically all identical except for the command. Another option would be to just have one, say calling it `db-script`. You could even set an `entrypoint` of `wait-for db:3306 --` and then your `run_inline` calls would just contain the specific command:
```
common.run_inline %W{docker-compose run --rm db-script ./generate-cdr/init-new-cdr-db.sh --cdr-db-name cdr}
common.run_inline %W{docker-compose run --rm db-script ./generate-cdr/init-new-cdr-db.sh --cdr-db-name public}

Reply:
Will consider this for a follow-up. The plus side is it makes docker-compose.yaml way simpler and makes things more direct; the downside is it means people issuing docker commands need to know what scripts to run.

Reply:
Cool.